### PR TITLE
STAND-77:Liquibase-demo-data.xml was not updated after changes made t…

### PR DESCRIPTION
**Pull request by gracebish**
**Pull request title**
STAND-77: Liquibase-demo-data.xml was not updated after changes made to tables
**Jira issue number and title**
STAND-77: Liquibase-demo-data.xml was not updated after changes made to tables
**Description of what i Changed:**
Update liquibase-demo-data.xml with the same changes that were made on liquibase-schema-only.xml
**Issue i worked on:**
https://issues.openmrs.org/browse/STAND-77

Checklist: I completed these to help reviewers :)

    [x] My IDE is configured to follow the code style of this project.

    No? Unsure? -> configure your IDE, format the code and add the changes with git add . && git commit --amend

    [x] I have added tests to cover my changes. (If you refactored
    existing code that was well tested you do not have to add tests)

    No? -> write tests and add them to this commit git add . && git commit --amend

    [x] I ran mvn clean package right before creating this pull request and
    added all formatting changes to my commit.

    No? -> execute above command

    [x] All new and existing tests passed.

    No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

    [x] My pull request is based on the latest changes of the master branch.

    No? Unsure? -> execute command git pull --rebase upstream master

### CODE REVIEW

**Comment by HerbertYiga** [comment]
@gracebish   could you add a ticket ID on this!!

**Comment by jwnasambu** [comment]
@gracebish LGTM congratulations.

**Comment by HerbertYiga** [comment]
@gracebish does this have a ticket ID,if so add it to your description

File: liquibase-demo-data.xml [hunk 1]
```diff
@@ -21,5 +21,13 @@
 			<column name="MD5SUM" type="varchar(35)" />
 		</ext:modifyColumn>
 	</changeSet>
-	   
+	
+	<changeSet id="drop-tribe-foreign-key-TRUNK-4492" author="dkayiwa">
```
Comment (slubwama) [inline liquibase-demo-data.xml:25]: @gracebish  is the author dkayiwa

Comment (gracebish) [reply liquibase-demo-data.xml:25]: Sure

Comment (gracebish) [reply liquibase-demo-data.xml:25]: @slubwama Did you see my last comment? It was based on https://github.com/openmrs/openmrs-core/commit/010be735a79865ed954a95400addf76ab8dfdb70#diff-7efe1c74929005aa8904f543f149c282

**Comment by gracebish** [comment]
@dkayiwa Kindly advise on this ticket.

**Comment by gracepotma** [comment]
@tendomart to have a look at this. Let's see if we can finish this ticket :)

**Comment by tendomart** [comment]
@gracebish were are you blocked here so that we can support you ?
Have you also seen how change can be propagated in the `liquibase-demo-data.xml`

like https://github.com/openmrs/openmrs-standalone/blob/master/liquibase-demo-data-18x.xml

**Pull request closed without merging**